### PR TITLE
Email editor extra tabs

### PR DESCRIPTION
**Pull request by ziggabyte**
## Description
This PR adds the Settings and Preview tabs. 

## Screenshots
bild
bild

## Changes
* Adds `Settings` and `Preview` tabs
* Adds fields to display (static) email address of org and name of org as sender, and an input for email subject line
* Adds UI for a button to send a test email to the current user

## Notes to reviewer
The button to send a test email does not do anything yet, it is only UI. Needs API to be finished to work fully.

## Related issues
none

### CODE REVIEW

File: src/features/emails/l10n/messageIds.ts [hunk 1]
```diff
@@ -5,6 +5,22 @@ export default makeMessages('feat.emails', {
     settings: {
       tabs: {
         content: m('Content'),
+        preview: {
+          emailInputError: m('This is not a valid email'),
+          emailLabel: m('Input your own email'),
```
Comment (richardolsson) [inline src/features/emails/l10n/messageIds.ts:10]: Are these used?

File: src/features/emails/hooks/useSendTestEmail.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { useState } from 'react';
+
+export default function useSendTestEmail() {
+  const [isLoading, setIsLoading] = useState(false);
+
+  return {
+    isLoading,
+    sendTestEmail: () => {
+      //TODO: real logic to send email here
```
Comment (richardolsson) [inline src/features/emails/hooks/useSendTestEmail.ts:9]: Great with the comment!

**Review by richardolsson — COMMENTED** [review]
Nice work! I have a question below before I'm ready to approve.

File: src/features/emails/l10n/messageIds.ts (hunk 1 above)
Comment (ziggabyte) [reply src/features/emails/l10n/messageIds.ts:10]: no! thank you for noticing!

**Pull request merged**